### PR TITLE
made create app button appear indepentent of app list load

### PR DESCRIPTION
**Pull request by oleterno-equinor**

### CODE REVIEW

File: src/components/app-list/index.js [hunk 1]
```diff
@@ -78,44 +78,50 @@ export class AppList extends React.Component {
...
+                  <Typography variant="body_short_bold">
+                    All applications
                   </Typography>
+                  <div className="app-list__list">{appsRender}</div>
                 </div>
-                <div>
-                  <PageCreateApplication />
+              </>
+            )}
+            {apps.length === 0 && (
```
Comment (Pespiri) [inline src/components/app-list/index.js:111]: suggestion: use `short-if`
```suggestion
            ) : (
```

File: src/components/app-list/index.js [hunk 2]
```diff
@@ -78,44 +78,50 @@ export class AppList extends React.Component {
...
       ) : (
-        <Typography variant="body_short">No favourites</Typography>
+        <Typography>No favourites</Typography>
       );
 
     return (
-      <article className="app-list">
+      <article className="grid grid--gap-medium">
+        <div className="app-list__header">
+          {apps.length ? (
```
Comment (Pespiri) [inline src/components/app-list/index.js:87]: suggestion: change for readability
```suggestion
          {apps.length > 0 ? (
```

File: src/components/app-list/index.js (hunk 1 above)
Comment (oleterno-equinor) [reply src/components/app-list/index.js]: fixed

File: src/components/app-list/index.js (hunk 2 above)
Comment (oleterno-equinor) [reply src/components/app-list/index.js]: fixed

**Review by Pespiri — APPROVED** [review]

**Pull request merged**